File: Python/876A-tripformeal.py
n = int(input())
a = int(input())
b = int(input())
c = int(input())
x = min(a, b, c)
if x == a or x == b or n == 1:
    print((n - 1) * min(a, b))
else:
    print(min(a, b) + c * (n - 2))

File: C++/A/876A-tripformeal.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    int n, a, b, c;
    cin >> n >> a >> b >> c;
    int x = min(a, min(b, c));
    cout << ((x == a || x == b || n == 1) ? min(a, b) * (n - 1) : min(a, b) + (c * (n - 2)));
}